fluent-syntax/test/fixtures_structure/whitespace_trailing.ftl

```diff
@@ -0,0 +1,5 @@
+#           < whitespace >
+key1 = Value              
+
+key2 = Value              {placeable}.
```

Comment:
I guess you didn't mean to have the trailing `.` here?

Reply:
Doch doch. I only meant to test that `TextElements` not in the last position are not affected by the trimming logic.